### PR TITLE
Add Golang SDK

### PR DESCRIPTION
**Pull request by xxx7xxxx**

### CODE REVIEW

File: go-sdk/stdlib/stdlib.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+	body, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		log.Printf("read config body failed: %v", err)
+	}
+
+	config := &AgentConfig{}
+	err = json.Unmarshal(body, config)
+	if err != nil {
+		log.Printf("unmarshal config body failed: %v", err)
+	}
```
Comment (localvar) [inline go-sdk/stdlib/stdlib.go:99]: `a.headers` should not be updated if there's an error.

Comment (xxx7xxxx) [reply go-sdk/stdlib/stdlib.go]: Good gotcha, updated.

**Review by localvar — APPROVED** [review]

File: go-sdk/stdlib/stdlib.go [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+	config := &AgentConfig{}
+	err = json.Unmarshal(body, config)
+	if err != nil {
+		log.Printf("unmarshal config body failed: %v", err)
+		return
+	}
+
+	headers := strings.Split(config.Headers, ",")
+	a.headers.Store(headers)
```
Comment (zhao-kun) [inline go-sdk/stdlib/stdlib.go:104]: Update only when the spec was changed?

Comment (xxx7xxxx) [reply go-sdk/stdlib/stdlib.go:104]: I have considered it, but it seems the operation to check if the config is changed won't be more efficient, moreover it requires to use more coarse-grained lock to compare-and-store. Considering that, I chose to simply store.

Comment (zhao-kun) [reply go-sdk/stdlib/stdlib.go:104]: It is fine with me

**Review by zhao-kun — APPROVED** [review]

**Pull request merged**